core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -389,10 +390,8 @@ def _ray_fit(
     fold_model.fit_time = time_train_end_fold - time_start_fold
     fold_model, pred_proba = _ray_predict_oof(fold_model, X_val_fold, y_val_fold,
                                               time_train_end_fold, resources['num_cpus'], save_bag_folds)
-    model_save_path = None
-    if model_sync_path is not None:
-        model_save_path = model_sync_path + f"{fold_model.name}/"
-    fold_model.save(model_save_path)
+    model_save_path = model_sync_path + f"{fold_model.name}/" if model_sync_path is not None else None
```

Comment:
`f"{fold_model.name}/"` , is there concern with cross-OS compatibility by the use of `/`?

I'd recommend at least adding a docstring for `model_sync_path` as it isn't immediately clear what this is doing.

Reply:
This path is a s3 path so we cannot use utils like `os.path.join`. I'll add a comment